.gitignore

```diff
@@ -6,4 +6,13 @@ node_modules/
 .DS_Store
 node_modules
 lib/forge-std/
-yarn-error.log
\ No newline at end of file
+yarn-error.log
+not_used
+solidity-flattener
+out/
+broadcast/
```

Comment:
remove this and the `!/broadcast` on line 15

Reply:
Done. I'll be commenting on all these at least with "done" to verify I have or will implement the change. Here, I've removed both those lines and left "/out/" and "/broadcast/" in the .gitignore.